app/views/layouts/_sidenav.html.erb

```diff
@@ -1,7 +1,7 @@
 <aside class="usa-width-one-fourth usa-layout-docs-sidenav">
   <ul class="usa-sidenav-list">
     <li>
-      <%= link_to 'Moving with the Military', page_path('moving-guide'), class: params[:id].include?('moving-guide') ? 'usa-current' : nil %>
+      <%= link_to 'Moving with the Military', page_path('moving-guide'), class: params[:id].try(:include?, 'moving-guide') ? 'usa-current' : nil %>
```

Comment:
`params[:id]` is `nil` on some pages which threw an error when trying to call `nil.include?`.